### PR TITLE
Sprockets and es6 Transpilation

### PR DESCRIPTION
**Pull request by joshstricklin**

### CODE REVIEW

**Review by vishalarora — COMMENTED** [review]
👍 Please fix the Hound complaints, but otherwise looks good to me.

**Review by vishalarora — APPROVED** [review]
☝️ same comment, just forgot to approve

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -14,3 +14,6 @@
 //= require jquery_ujs
 
 //= require_tree .
+
+let name = "josh"
```
[the remark this answers is not in the document]
Comment (wontoan) [reply app/assets/javascripts/application.js:18]: :O

[the remark this answers is not in the document]
Comment (joshstricklin) [reply app/assets/javascripts/application.js:18]: Hey, dude

**Pull request merged**